src/components/manage/Blocks/Image/ImageSidebar.jsx

```diff
@@ -19,50 +21,54 @@ const ImageSidebar = (props) => {
       </header>
 
       <Segment className="sidebar-metadata-container" secondary attached>
-        {typeof data.url === 'string' ? (
-          // Entered an external URL
-          <>
-            {data.url.split('/').slice(-1)[0]}
+        <>
+          <div>
+            {(data.url?.['@id'] ?? data.url).split('/').slice(-1)[0]}
+            <Button
+              basic
+              className="cancel"
+              onClick={(e) => {
+                e.stopPropagation();
+                onChangeBlock(block, {
+                  ...data,
+                  url: undefined,
+                  image_scales: undefined,
+                });
+              }}
+            >
+              <Icon name={clearSVG} size="30px" />
```

Comment:
This still needs to be styled properly, but the point is to add a way to remove the currently selected image.